__tests__/actionCreators.test.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import { authenticationFailed, authenticationSucceeded, signedOut } from '../src/actionCreators';
+
+describe('authenticationFailed', () => {
+  const currentUser = { hello: 'world' }
+  const errInfoauthenticate = { foo: 'bar' }
+
+  const authResult = {
+    currentUser: currentUser,
+    authenticationError: errInfoauthenticate
+  };
```

Comment:
OMG - the dreaded semicolon!   (slightly prefer without them; not a biggie)

Reply:
Should we add that to es-lint?

Reply:
I added a change to eslint for this: #590